### DISCUSSION TITLE
Welcome to FlappyBird Discussions!

### DISCUSSION THREAD

**Discussion by VadimBoev** [VadimBoev/FlappyBird#32] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SRC267** [comment]
Just sharing my high score: ![Screenshot_20241007-035259.png](https://github.com/user-attachments/assets/5515c598-1a54-4584-9fd1-5c1b2b70eb85)

**Comment by 004Ongoro** [comment]
How did you achieve that. That's the hardest game to play.

**Comment by SRC267** [comment]
> How did you achieve that. That's the hardest game to play.

Have you tried the game? I feel like this version is much easier than the original. I haven't played much since, and I'm sure I'll easily pass that score if I try lol.

**Reply by VadimBoev** [reply]
> > How did you achieve that. That's the hardest game to play.
> 
> Have you tried the game? I feel like this version is much easier than the original. I haven't played much since, and I'm sure I'll easily pass that score if I try lol.

Yes, you're right, this version is smoother.
